src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitService.kt

```diff
@@ -236,6 +238,12 @@ class VisitService(
           groupLeader = false,
         )
       }
+
+      if (voVisitorsToRemove.isNotEmpty()) {
+        // wait for any locks in NOMIS
```

Comment:
Better to have the word "lock" in the method name rather than this comment? maybe `lockAllByIdIn`.

Reply:
Hmm, doesn't it have to be the Spring Data name given there is no HQL?